shared/react-native/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -69,6 +71,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
 
+        if (getIntent().getExtras().containsKey("notification")) {
+            ReactContext currentContext = getReactInstanceManager().getCurrentReactContext();
+            if (currentContext != null) {
+                currentContext
+                        .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
+                        .emit("androidIntentNotification", "");
```

Comment:
`null` better than `""`? https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/core/DeviceEventManagerModule.java seems to indicate `null` is ok